sdk/rust/oak/src/lib.rs

```diff
@@ -308,10 +308,14 @@ pub fn random_get(buf: &mut [u8]) -> OakStatus {
 /// [panic information]: std::panic::PanicInfo
 pub fn set_panic_hook() {
     std::panic::set_hook(Box::new(|panic_info| {
-        let msg = panic_info
-            .payload()
-            .downcast_ref::<&str>()
-            .unwrap_or(&"<UNKNOWN MESSAGE>");
+        let payload = panic_info.payload();
```

Comment:
Could you add a link in a comment to the upstream one? Would be useful as a reference if things change in the future.

Reply:
Done